challenge_5/python/Loran425/src/FindTheDifference.py

```diff
@@ -0,0 +1,23 @@
+__author__ = "Loran425"
+
+
+class Solution(object):
+    def findTheDifference(self, s, t):
+        count = {}
```

Comment:
There's a [Counter](https://docs.python.org/2/library/collections.html#collections.Counter) class for this. May be useful in the future.